index.html

```diff
@@ -14,6 +14,56 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header">
+      
+        <nav class="nav_link">
+            <img src="./img/karma-logo.svg" alt="">
```

Comment:
```suggestion
            <img src="./img/karma-logo.svg" alt="Karma logo">
```
Images should have an `alt` description for screen reader users. Same goes for the other `img` elements in this file